src/port1.0/portbump.tcl

```diff
@@ -0,0 +1,208 @@
+# et:ts=4
```

Comment:
The modeline seems incomplete to me. Did you partially copy this line from a different file?

Reply:
Yes, but it's an exact copy from `portchecksum.tcl`. It's also present in some other files like `portinstall.tcl`. Some files are more whole tho like this:
```suggestion
# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```
Should we use that instead? These are used in traditional editors like vim right?

Reply:
Yes, the modeline described in [HACKING](https://github.com/macports/macports-base/blob/master/HACKING) should be used.